tests/test_amity.py

```diff
@@ -0,0 +1,62 @@
+import unittest
+from amity import Amity
+
+
+class AmityTest(unittest.TestCase):
+
+    def setUp(self):
+        self.building = Amity()
+
+    def test_populate_offices(self):
+        '''
+        Test whether the populate method
+        populates the building with office rooms
+        '''
+        self.building.populate()
+        self.assertTrue(len(self.building.office_rooms) > 0)
```

Comment:
**Code Quality: Nitpick**

Would be nice to ensure that the `office_rooms` list is empty before the `populate` function is called.